package.json

```diff
@@ -107,12 +115,14 @@
     "verdaccio-theme"
   ],
   "scripts": {
+    "type-check": "tsc --noEmit",
+    "type-check:watch": "npm run type-check -- --watch",
     "flow": "flow check",
     "release": "standard-version -a -s",
     "test:clean": "npx jest --clearCache",
     "test": "cross-env NODE_ENV=test BABEL_ENV=test TZ=UTC jest --config ./jest.config.js --maxWorkers 2 --passWithNoTests",
     "test:size": "bundlesize",
-    "lint": "npm run flow && npm run lint:js && npm run lint:css",
+    "lint": "npm run type-check && npm run lint:js && npm run lint:css",
```

Comment:
The linting must use https://github.com/verdaccio/eslint-config-verdaccio
you can follow the @iHDeveloper recent migration about how to do it.
https://github.com/verdaccio/verdaccio-google-cloud/blob/master/.eslintrc